ibis/config.py

```diff
@@ -174,7 +174,26 @@ def _default_backend() -> Any:
     try:
         import duckdb as _  # noqa: F401
     except ImportError:
-        return None
+        raise ImportError(
+            """
```

Comment:
So the error message doesn't start with a newline

```suggestion
            """\
```